applications/mne_anonymize/mainwindow.h

```diff
@@ -249,7 +249,7 @@ class MainWindow : public QMainWindow
      * @param[in] s Absolute path of the input file.
      *
      */
-    void fileInChanged(const QString& s) const;
```

Comment:
@juangpc Can you elaborate why you feel the const qualifier is redundant here? I always find it nice to read which functions do not change the internal state of an instantiated object.

Reply:
Because they're signals.